src/connectors/range-slider/connectRangeSlider.js

```diff
@@ -101,26 +101,33 @@ export default function connectRangeSlider(renderFn) {
         };
       },
 
-      getConfiguration: originalConf => {
-        const conf = {
+      getConfiguration: currentConfiguration => {
+        const configuration = {
           disjunctiveFacets: [attributeName],
         };
 
-        const hasUserBounds = userMin !== undefined || userMax !== undefined;
+        const isMinBound = userMin !== undefined && userMin !== null;
+        const isMaxBound = userMax !== undefined && userMax !== null;
+        const boundsDefined = isMinBound || isMaxBound;
```

Comment:
I think we should be able to refactor more and use this:

```js
const boundsDefined = isSafeInteger(userMin) || isSafeInteger(userMax);
```
http://devdocs.io/lodash~4/index#isSafeInteger

Reply:
Ok for me. 
But if the user set min / max with a floating number `isSafeInteger` will fail. 
Maybe we should go for `isFinite` ? Or we consider that passing floating numbers for bounds is not valid.